### ISSUE TITLE
Add more information on the authentication server

### ISSUE THREAD

**Issue by okdistribute** [getditto/docs#547]
* Flesh the authentication server with real running code
* How to test that your auth server is working? 
   * Copy/paste curl requests
   * Troubleshooting the token parsing
   * JSON stringify 
* Examples to deploy with AWS and Google cloud (Deploy server)
    * Deploy publicly to internet
    * Include screenshots or specific parts of the cloud console

https://docs.ditto.live/android/common/security/authentication#server

**Comment by contenuto** [comment]
This is what we have for auth server:
- [C# > Authentication Server: C# article]([https://docs.ditto.live/csharp/authentication-server]
- [A blurb in Troubleshooting.](https://docs.ditto.live/troubleshooting)

Also, how can we go about getting a diagram showing the client- and server-side auth flow? @stusmall, is this something you can help me with?

**Comment by stusmall** [comment]
Absolutely.  It's on my short term to-do list to create more internal documentation around this for our own uses.  Portions of this will also be valuable as public, customer facing documentation

**Comment by okdistribute** [comment]
The material is here: https://docs.ditto.live/security/onlinewithauthentication
